sphinx/util/nodes.py

```diff
@@ -324,24 +325,20 @@ def traverse_translatable_index(
         yield node, entries
 
 
-def nested_parse_with_titles(state: Any, content: StringList, node: Node,
+def nested_parse_with_titles(state: RSTState, content: StringList, node: Node,
                              content_offset: int = 0) -> str:
     """Version of state.nested_parse() that allows titles and does not require
     titles to have the same decoration as the calling document.
 
     This is useful when the parsed content comes from a completely different
     context, such as docstrings.
+
+    This function is retained for compatability and will be deprecated in
+    Sphinx 8. Prefer ``parse_block_text()``.
```

Comment:
This refers to a function that does not exist?
Are you intending to implement it?
Also if it is to be deprecated, then it should emit a deprecation warning when called no?

Reply:
A reference to an old name, will update.

A